Source/ACE/Entity/Landblock.cs

```diff
@@ -19,6 +19,11 @@
 
 namespace ACE.Entity
 {
+    using global::ACE.Entity.Enum.Properties;
+    using global::ACE.Network.Sequence;
+
+    using static LandblockManager;

```

Comment:
You used some automatic tool that did the "using static" stuff which is new in .Net 6 and that quite frankly, I absolutely think is a bad thing they did.  It's akin to VB6's "with" statement, only scoped to the entire file.